src/applications/hca/definitions/dependent.js

```diff
@@ -17,7 +16,7 @@ const {
   dependents: {
     items: { properties: dependent },
   },
-} = fullSchemaHca.properties;
```

Comment:
Did we drop the `hca` from this name since we are in the `hca` app folder already?

Reply:
Correct. It seems a bit redundant.